docs/install/macos.mdx

```diff
@@ -9,99 +9,71 @@ title: Build on MacOS
 In order to compile PcapPlusPlus on MacOS please make sure you have the following components installed:
 
 1. [Xcode](https://developer.apple.com/xcode/) which contains all prerequisites required for PcapPlusPlus including gcc/g++ compiler and libpcap with all relevant header files
-2. Make sure you have Xcode Command Line Tools. You can install it by running the following command in Terminal:
-
-   ```shell
-   xcode-select --install
```

Comment:
I think we can leave this. Xcode will also provide clang compiler also.

Reply:
Note 100% sure

Reply:
Yeah I'm also not sure, let's leave it as it is